app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -17,6 +17,107 @@ const logger = new Logger('CustomOAuth');
 const Services = {};
 const BeforeUpdateOrCreateUserFromExternalService = [];
 
+const normalizers = {
```

Comment:
I'm not sure if the javascript guarantee any kind of order for this case, maybe use a Map instead.

Reply:
It doesn't need to be in order